deepchem/molnet/load_function/tests/test_swissprot_loader.py

```diff
@@ -0,0 +1,26 @@
+"""
+Tests for swissprot loader.
+"""
+
+import os
+import numpy as np
+
+#from deepchem.molnet import load_swissprot
+from swissprot_datasets import load_swissprot
+@slow
+def test_swissprot_loader():
+  current_dir = os.path.dirname(os.path.abspath(__file__))
+  codes = ['A', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L',
+         'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'V', 'W', 'Y']
+  tasks, datasets = load_swissprot(
+      reload=False,
+      data_dir=current_dir,
+      featurizer=dc.featurizer.OneHoteFeatureizer(codes),
```

Comment:
Typo here. Should be `OneHotFeaturizer`. Can you verify that the unit test passes locally?

Reply:
The unit test passes locally. Other formatting problems were already solved